### ISSUE TITLE
[Docs] The pretrained weight of Matting-DIM

### ISSUE THREAD

**Issue by caihaunqai** [open-mmlab/mmediting#1453]
### Branch

master branch  https://mmediting.readthedocs.io/en/latest/

### 📚 The doc issue

Hello, thank you for your excellent job. When using Deep Image Matting, I notice that the channel of vgg16's first layer is 4, which combines the RGB and trimap channels. However, the pre-trained weight's first layer channel is 4 instead of 3.  How is this pre-training weight trained? As far as I understand, through imagenet training, the input graph should only have three channels. Hope for your reply. Thank you.
